src/main.cc

```diff
@@ -819,6 +819,11 @@ int main(int argc, char* argv[])
         {
             if (not name.empty() and name[0_byte] == '+')
             {
+                if (name.begin()+1 == name.end() or *(name.begin()+1)  == ':')
```

Comment:
Yeah, or just compare the whole string: `if (name == "+" or name == "+:")` will be more explicit, and this is not performance critical code.